src/api/google-drive-api.js

```diff
@@ -214,3 +215,15 @@ function findNotesByName(name = '') {
     });
   });
 }
+
+function findNotesById(lastViewedNoteId) {
```

Comment:
Note ID is unique. How can we find multiple notes by ID?

Let's rename it to `findNoteById` (without `S`) ?